### PR TITLE
[13.0][MIG] mrp_bom_structure_xlsx

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
mrp_bom_structure_xlsx migration to v13.

Depends on https://github.com/OCA/reporting-engine/pull/331

cc ~ @ForgeFlow @LoisRForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
functionally and technical review locally :+1:

**Review by HviorForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

File: mrp_bom_structure_xlsx/__manifest__.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
+# Copyright 2017-19 ForgeFlow S.L. (https://www.forgeflow.com)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "MRP BOM Structure XLSX",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Manufacturing",
     "summary": "Export BoM Structure to Excel .XLSX",
     "author": "Eficent, Odoo Community Association (OCA)",
```
Comment (AdriaGForgeFlow) [inline mrp_bom_structure_xlsx/__manifest__.py]: Could you update the author?

**Review by AdriaGForgeFlow — COMMENTED** [review]
Code review :+1: 

Just small comment above

**Review by AdriaGForgeFlow — APPROVED** [review]

**Pull request merged**